src/components/stepforms/widgets/DateComponents/NewDateInput.vue

```diff
@@ -35,6 +35,7 @@
           :availableVariables="availableVariables"
           :selectedVariables="selectedVariables"
           :enableCustom="enableCustom"
+          :showOnlyLabel="!isEditorOpened"
```

Comment:
So the value of variables is displayed when we open the dropdown, but then hides when "custom" is selected?
Neat!

Reply:
Yes it avoid the strange behaviour of tooltip going to top left screen when closing the popover :wink: